osmaxx-py/osmaxx/excerptexport/models/extraction_order.py

```diff
@@ -51,6 +51,7 @@ class ExtractionOrder(models.Model):
     orderer = models.ForeignKey(User, related_name='extraction_orders', verbose_name=_('orderer'))
     excerpt = models.ForeignKey(Excerpt, related_name='extraction_orders', verbose_name=_('excerpt'))
     progress_url = models.URLField(verbose_name=_('progress URL'), null=True, blank=True)
+    process_start_time = models.DateTimeField(verbose_name=_('process start time'), null=True, blank=True)
```

Comment:
instead of blank, we could also use [auto_now_add](https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.DateField.auto_now_add) - then it wont be set from inside a view, but automatically, which might or might not be the desired behavior.